doc/reference/networking/net_pkt.rst

```diff
@@ -10,6 +10,355 @@ Packet Management
 Overview
 ********
 
+Network packets are the main data the networking stack manipulates.
+Such data is represented through the net_pkt structure which provides
+a mean to hold the packet, write and read it, as well as necessary
```

Comment:
>  It might be better to do your own editing based on these suggested edits and submit one amended commit rather than using the GitHub accept suggestion button.

BTW, at least I have never pressed the "Commit suggestion" button, instead I am always editing my local branch as in many times I need to reformat the lines (to make them shorter etc). And it also feels more kosher thing to do anyway :)